### ISSUE TITLE
'Scanning your file...' never completes because of "Script execution took more than 30 seconds."

### ISSUE THREAD

**Issue by konstan** [bridgecrewio/checkov-jetbrains-ide#93]
How to increase the timeout on the plugin in IntelliJ?

I see the following in the logs of IntelliJ

```
2022-03-17 11:18:40,425 [1743730]   INFO - ew.services.CheckovScanService - Trying to scan a file using com.bridgecrew.services.checkovService.DockerCheckovService@a618bb6 
2022-03-17 11:18:40,427 [1743732]   INFO - ew.services.CheckovScanService - Running command: docker run --rm --tty --env BC_SOURCE=jetbrains --env BC_SOURCE_VERSION=0.0.17 --env LOG_LEVEL=DEBUG --volume xxx:/checkovScan/es-deployment.yaml bridgecrew/checkov -d /checkovScan -s --bc-api-key **-**-**-** --repo-id jetbrains/extension -o json 
2022-03-17 11:19:10,454 [1773759]  ERROR - intellij.openapi.progress.Task - Script execution took more than 30 seconds. 
com.intellij.execution.ExecutionException: Script execution took more than 30 seconds.
	at com.intellij.execution.process.ScriptRunnerUtil.getProcessOutput(ScriptRunnerUtil.java:67)
	at com.bridgecrew.services.CheckovScanService$ScanTask.run(CheckovScanService.kt:162)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:962)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:472)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:235)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:178)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:658)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:610)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:65)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:165)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:235)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-03-17 11:19:10,467 [1773772]  ERROR - intellij.openapi.progress.Task - IntelliJ IDEA 2020.3.3 
2022-03-17 11:19:10,468 [1773773]  ERROR - intellij.openapi.progress.Task - JDK: 11.0.10; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2022-03-17 11:19:10,468 [1773773]  ERROR - intellij.openapi.progress.Task - OS: Mac OS X 
2022-03-17 11:19:10,469 [1773774]  ERROR - intellij.openapi.progress.Task - Plugin to blame: Checkov version: 0.0.17 
```

Indeed, the command takes ~55 sec when run from CLI

```
$ time docker run --rm --tty --env BC_SOURCE=jetbrains --env BC_SOURCE_VERSION=0.0.17 --env LOG_LEVEL=DEBUG --volume xxx:/checkovScan/es-deployment.yaml bridgecrew/checkov -d /checkovScan -s --bc-api-key xxx-key --repo-id jetbrains/extension
...
real    0m55.472s
user    0m0.110s
sys     0m0.074s
$
```

**Comment by orhovy** [comment]
Hi @konstan , Thanks for bringing that up, I will update the scan timeout and this change will be deployed in the next version

**Comment by orhovy** [comment]
Closing this issue as a fix was deployed to version 0.0.18
Please update otherwise.
